ui/src/dms/NewDm.tsx

```diff
@@ -21,7 +22,29 @@ export default function NewDM() {
   const navigate = useNavigate();
   const shipValues = useMemo(() => ships.map((o) => preSig(o.value)), [ships]);
   const newClubId = useMemo(() => newUv(), []);
-  const sendMultiDm = useSendMultiDm(true, shipValues);
+  const existingMultiDm = useMemo(() => {
+    const { briefs } = useChatState.getState();
+    return Object.entries(multiDms).reduce<string>((key, [k, v]) => {
+      const theShips = [...v.hive, ...v.team];
+      const sameDM = _.difference(shipValues, theShips).length === 0;
+      const brief = briefs[key];
+      const newBrief = briefs[k];
+      const newer = !brief || (brief && newBrief && newBrief.last > brief.last);
+      if (sameDM && newer) {
+        return k;
+      }
+
+      return key;
+    }, '');
+  }, [multiDms, shipValues]);
+  const who =
+    ships.length > 0
+      ? isMultiDm
+        ? existingMultiDm
+          ? existingMultiDm
```

Comment:
Confirming this nested ternary checking for `existingMultiDm` in sequence is expected?

Reply:
Hmm this probably could be simplified